src/client/views/cube-view/api-context.tsx

```diff
@@ -97,6 +102,10 @@ function createVizQueryApi(settings: ClientAppSettings): VisualizationQuery {
   return createApiCall(settings, "visualization", emptyParams);
 }
 
+function createNumberFilterQuery(settings: ClientAppSettings): BooleanFilterQuery {
```

Comment:
Shouldn't the return type be actually a `NumberFilterQuery`?

Reply:
😱